propulsion-cosmos-reactor/Ingester.fs

```diff
@@ -0,0 +1,47 @@
+module ReactorTemplate.Ingester
```

Comment:
I find it works well to call this file Reactor.fs (dont forget to rename the module)